lib/ignore.js

```diff
@@ -28,6 +28,11 @@ function ignore(dir, options) {
       // add the xpi file name to the ignore list
       var xpiName = getID(manifest) + "*.xpi";
       defaultRules.push(xpiName);
+      // add the signed xpi file name to the ignore list
+      var addonTitle = manifest.title || manifest.name || "Untitled";
+      var signedXpiName = addonTitle.toLowerCase().replace(/ /g, "_") + "*.xpi";
```

Comment:
This looks like it will work. If you want to trace the AMO code, this is how it generates filenames https://github.com/mozilla/addons-server/blob/master/src/olympia/files/models.py#L239